cocotb/share/include/embed.h

```diff
@@ -32,6 +32,7 @@
 
 #include <gpi.h>
 #include <gpi_logging.h>
+#include <vpi_user_ext.h>
```

Comment:
How is this file named in Cadence tools? I'm wondering if it should be `sv_vpi_user_ext.h` instead, since we now always include sv_vpi_user.h (which in turn includes the older vpi_user.h).

Reply:
It's called ``vpi_user_cds.h``. 
I guess if we get more defines from other simulators, we will handle that with ifdef inside the same file if needed.